xcm/src/v1/README.md

```diff
@@ -1,25 +0,0 @@
-# XCM Version 1
-The comprehensive list of changes can be found in [this PR description](https://github.com/paritytech/polkadot/pull/2815#issue-608567900).
-
-## Changes to be aware of
-Most changes should automatically be resolved via the conversion traits (i.e. `TryFrom` and `From`). The list here is mostly for incompatible changes that result in an `Err(())` when attempting to convert XCM objects from v0.
-
-### Junction
```

Comment:
why remove this file?

Reply:
There was a discussion where we agreed that we shouldn't maintain duplicated information on 2 different locations (i.e. both on the README.md and the top-level doc comments for the Rust module), and so I opted for only writing it on the top-level doc comments.